File: TumCampusApp/src/de/tum/in/tumcampusapp/models/TelSubstation.java
package de.tum.in.tumcampusapp.models;

import java.io.Serializable;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

/**
 * Telefon substation to reach an employee. Note: This model is based on the TUMOnline web service response format for a
 * corresponding request.
 * 
 * @author Vincenz Doelle
 * @review Daniel G. Mayr
 */
@Root(name = "nebenstelle", strict = false)
public class TelSubstation implements Serializable {

	private static final long serialVersionUID = -3289209179488515142L;

	@Element(name = "telefonnummer")
	private String number;

	public String getNumber() {
		return number;
	}

	public void setNumber(String number) {
		this.number = number;
	}
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/person/model/TelSubstation.kt
package de.tum.`in`.tumcampusapp.component.tumui.person.model

import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml
import java.io.Serializable

/**
 * Telefon substation to reach an employee. Note: This model is based on the
 * TUMOnline web service response format for a corresponding request.
 */
@Xml(name = "nebenstelle")
data class TelSubstation(@PropertyElement(name = "telefonnummer")
                         var number: String = "") : Serializable {
    companion object {
        private const val serialVersionUID = -3289209179488515142L
    }
}
